crystal/distribution.yaml

```diff
@@ -1714,6 +1714,21 @@ repositories:
       url: https://github.com/ros2/teleop_twist_keyboard.git
       version: ros2
     status: maintained
+  tinydir_vendor:
+    doc:
+      type: git
+      url: https://github.com/aws-robotics/vendored-tinydir.git
```

Comment:
If this is going to be depended on by `rcl`, then I think it would make more sense to host this under the `ros2` organization.  @ros2/team, thoughts?

Reply:
That seems reasonable to me. If we move it we should also move the release repository under ros2-gbp and make sure the maintainers still have access to both.

Reply:
We talked about this internally and came to the conclusion that it makes sense to host all repositories containing packages for the core of ROS 2 under the `ros2` / `ros2-gbp` org units.

@AAlon Can you move both repositories to my GitHub user and then I will move them into the ROS 2 org units as well as grant you full access to both repos again. If there are any other GitHub users which I should add initially please let me know - otherwise any admin can add them later.

Reply:
New location: https://github.com/ros2-gbp/tinydir_vendor-release

Reply:
> Regarding permissions, if you could add everyone under https://github.com/orgs/aws-robotics/people that would be great

The list of users is not publicly accessible: "This organization has no public members."

Reply:
> if could add everyone under https://github.com/orgs/aws-robotics/people that would be great.,

@AAlon you should have the permissions to add others as collaborators to the repo.

Reply:
Could you also grant me the permissions for the source repo? (https://github.com/ros2/tinydir_vendor)
Thanks!